### ISSUE TITLE
Build error : Expiring Daemon because JVM heap space is exhausted in version 11.0.2

### ISSUE THREAD

**Issue by duongbi** [dpa99c/cordova-plugin-firebasex#538]
I build cordova project with cordova android 9.0.0 and  cordova-plugin-firebasex 11.0.2 always error: Expiring Daemon because JVM heap space is exhausted and it takes over 30 minutes.
With cordova-plugin-firebasex 3.0.8 it takes only 10 seconds to build.
Please tell me what happen with new version!

**Comment by trung-pham** [comment]
same issue here

**Comment by dpa99c** [comment]
> error: Expiring Daemon because JVM heap space is exhausted

This indicates Java is running out of memory when building your Android app.

> With cordova-plugin-firebasex 3.0.8 it takes only 10 seconds to build.

Since then, this plugin has added support for new Firebase SDK components and also newer versions of previously supported Firebase SDK components have grown in size as Google adds more functionality to Firebase.
This means when you include recent versions of this plugin into your Cordova project, Android builds require more memory than earlier versions of the plugin.

TL;DR: this is not a plugin bug but an issue with memory allocation to Java in your development environment.

If you allocate more memory to the JVM, the issue will go away - [see here](https://stackoverflow.com/questions/56075455/expiring-daemon-because-jvm-heap-space-is-exhausted) for examples of how to do this.

**Comment by adelalmaleh2** [comment]
Maybe you could add 
`org.gradle.daemon=true`
`org.gradle.jvmargs=-Xmx2560m`
into gradle.properties file

**Comment by ShashankHegde16** [comment]
org.gradle.daemon=true
org.gradle.jvmargs=-Xmx2560m. this also not working.

**Comment by MaximBelov** [comment]
try remove com.google.firebase:firebase-perf 
https://github.com/MaximBelov/cordova-plugin-firebasex/commit/7d2e7ff64269f4184328e9d56b6ef8b6e6193706
